specs/ecosystem/ecosystem.spec

```diff
@@ -281,5 +281,21 @@ Steps:
       |----|-----------------|----------|-----------------------|
       |1   |maven-run        |successful|no                     |
 
-## need to add from clustertasks  
-## s2i, kn, kn-apply, tkn-pac, to be added.
\ No newline at end of file
+## Test the functionality of step action resolvers: PIPELINES-29-TC14
+Tags: e2e, sanity, ecosystem, non-admin
+Component: Resolvers
+Level: Integration
+Type: Functional
+Importance: High
+
+Steps:
+    * Verify ServiceAccount "pipeline" exist
+    * Create 
+      |S.NO|resource_dir                                                     |
+      |----|-----------------------------------------------------------------|
+      |1   |testdata/ecosystem/tasks/git-clone-stepaction.yaml               |
+      |2   |testdata/ecosystem/pipelineruns/git-clone-stepaction.yaml        |
```

Comment:
Pipeline run is stuck
```
0/5 nodes are available: persistentvolumeclaim "shared-pvc" not found. preemption: 0/5 nodes are available: 5 Preemption is not helpful for scheduling.
```

Reply:
Done
`2024/11/28 10:32:08 Waiting for operator to be up and running....
2024/11/28 10:32:09 Operator is up
      * Validate Operator should be installed	 ...[PASS]
2024/11/28 10:32:09 Verify SA "pipeline" is created in namespace "releasetest-rx4j6"
      * Verify ServiceAccount "pipeline" exist	 ...[PASS]
2024/11/28 10:32:11 output: task.tekton.dev/git-clone-stepaction created
2024/11/28 10:32:12 output: pipelinerun.tekton.dev/git-clone-stepaction-run created
2024/11/28 10:32:13 output: persistentvolumeclaim/shared-pvc created
      * Create 
      
         |S.NO|resource_dir                                             |
         |----|---------------------------------------------------------|
         |1   |testdata/ecosystem/tasks/git-clone-stepaction.yaml       |
         |2   |testdata/ecosystem/pipelineruns/git-clone-stepaction.yaml|
         |3   |testdata/pvc/pvc.yaml                                    |	 ...[PASS]
2024/11/28 10:32:13 validating pipeline run git-clone-stepaction-run for success state...
2024/11/28 10:32:48 pipelineRun: git-clone-stepaction-run is successful under namespace : releasetest-rx4j6
      * Verify pipelinerun 
      
         |S.NO|pipeline_run_name       |status    |check_label_propagation|
         |----|------------------------|----------|-----------------------|
         |1   |git-clone-stepaction-run|successful|no                     |	 ...[PASS]
2024/11/28 10:32:56 output: project.project.openshift.io "releasetest-rx4j6" deleted

Successfully generated xml-report to => /Users/jayesh/project/release-tests/reports/xml-report

Transformed SuiteResult to report structure
Successfully generated html-report to => /Users/jayesh/project/release-tests/reports/html-report/index.html

Done generating HTML report using theme from /Users/jayesh/.gauge/plugins/html-report/4.3.1/themes/default
Sending kill message to Html Report plugin.
Sending kill message to Xml Report plugin.
Sending kill message to runner.
ok  	command-line-arguments	54.703s
Runner with PID:30798 has exited
Specifications:	1 executed	1 passed	0 failed	0 skipped
Scenarios:	1 executed	1 passed	0 failed	0 skipped

Total time taken: 52.039s
Updates are available. Run `gauge update -c` for more info.`

Reply:
While this will work, it is not in logical order... create PVC before creating pipelinerun

Reply:
Done